src/DependencyInjection/Configuration.php

```diff
@@ -49,6 +49,16 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('system_tasks')
+                    ->prototype('array')
+                        ->children()
+                            ->booleanNode('enabled')->defaultTrue()->end()
```

Comment:
Wouldn't `false` be a better default? Or does it still work if the three settings below are not set?